.github/workflows/windows/prepare-build/action.yml

```diff
@@ -19,7 +19,7 @@ runs:
   steps:
     - name: Setup PHP SDK
       id: setup-php
-      uses: cmb69/setup-php-sdk@v0.7
+      uses: alcaeus/setup-php-sdk@php-8.3
```

Comment:
Based on the [Room11 transcript](https://chat.stackoverflow.com/transcript/11?m=56782751#56782751), I assume we should just use your fork going forward until this gets moved into the PHP organization.